draft-wang-ppm-dap-taskprov.md

```diff
@@ -296,6 +296,23 @@ A protocol participant MUST opt out if the task has expired.
 The behavior of each protocol participant is determined by whether or not they
 opt in to a task.
 
+## Supporting a Global HPKE Configuration {#global-hpke-config}
+
+In DAP, Clients need to know the HPKE configuration of each Aggregator before
+sending reports (See HPKE Configuration Request in
+{{!DAP=I-D.draft-ietf-ppm-dap-02}}.)
+
+If `task_prov` extension is supported in a DAP deployment, each Aggregator
+SHOULD support a global HPKE configuration independent of Task ID, because
+a task is only configured in an Aggregator after the Aggregator has received
+the first Client report that contains the `task_prov` extension.
+
+If a Client wants to query an Aggregator's `[aggregator]/hpke_config` endpoint,
```

Comment:
There is no reason to repeat the endpoint format here. The more we can refer to the DAP draft as the source of truth, the better.